inst/book/doublet-detection.Rmd

```diff
@@ -181,13 +181,14 @@ This will use the spike-in size factors to scale the contribution of each cell t
 
 ### Doublet classification  
 
-The `scDblFinder()` function [@germain2020pipecomp] implements a related approach based on doublet simulation.
+The `scDblFinder()` function [@germain2021scDblFinder] combines this artificial doublet density approach with an iterative classification scheme. 
 For each observed cell, an initial score is computed by combining the fraction of simulated doublets in its neighborhood with another score based on co-expression of mutually exclusive gene pairs [@bais2020scds].
 A threshold is chosen that best distinguishes between the real and simulated cells, allowing us to obtain putative doublet calls among the real cells.
 The threshold and scores are then iteratively refined by training a classifier on the putative calls with a variety of metrics to characterize the doublet neighborhood.
 These metrics include the low-dimensional embeddings in PC space, the fraction of doublets among the $k$ nearest neighbors for a variety of $k$,
 the distance to the closest real cell, the expected within-cluster doublet formation rate, the observed number of cells in each cluster, and so on.
 The classifier distills all of these metrics into a single score based on their learnt importance (Figure \@ref(fig:scdblfinder-tsne)).
+The method can optionally use clusters for the generation of artificial doublets:
```

Comment:
Try to avoid colons, there's no guarantee that the plot will actually render below the text in other formats. (Not sure if I always follow that rule myself, but we should try in case someone wants a PDF.) I would say something like:

> The method can optionally use clustering information to guide the proportional generation of artificial doublets via the `clusters=` argument.

Can't recall if that's exactly what the `clusters=` does, but you get the idea.